char command
int64 modifier
---
int64 response_code
